### ISSUE TITLE
Xpand latest version - Unable to create an instance of the "Xpand.ExpressApp.Email.EmailModule" type.

### ISSUE THREAD

**Issue by david0villa** [eXpandFramework/eXpand#718]
Hi,

After upgrading to Xpand latest version - 20.1.401.0 and I did encounter the below issue with xafml file. Would you please advise me the fix

`at DevExpress.ExpressApp.ApplicationModulesManager.AddModule(ModuleBase module)
   at DevExpress.ExpressApp.ApplicationModulesManager.AddModule(Type moduleType, Boolean loadModuleDiffs)
   at DevExpress.ExpressApp.Utils.DesignerModelFactory.CreateModulesManager(ModuleBase module, String assembliesPath, List`1 additionalModules, ISecurityStrategyBase securityInstance)
   at DevExpress.ExpressApp.Design.ModelEditor.ModelLoader.LoadModel(ITypeDiscoveryService typeDiscoveryService, ITypeResolutionService typeResolutionService, String targetDiffFileName, IDisposable& obj)
   at DevExpress.ExpressApp.Design.ModelEditor.EditorPane.CreateModelEditorController()
   at DevExpress.ExpressApp.Design.ModelEditor.EditorPane.ReInit()
----------InnerException----------
Unable to create an instance of the "Xpand.ExpressApp.Email.EmailModule" type.
Reason: "The type initializer for 'Xpand.Persistent.Base.General.XpandModuleBase' threw an exception."
----------StackTrace----------
   at DevExpress.ExpressApp.Utils.Reflection.TypeData.CreateInstance(Object[] args)
   at DevExpress.ExpressApp.ModuleBase.CreateModule(Type moduleType, ModelStoreBase diffsStore)
   at DevExpress.ExpressApp.Core.ModuleFactory.ModuleFactoryWithResourcesDiffsStore.CreateModule(Type moduleType)
   at DevExpress.ExpressApp.ApplicationModulesManager.EnsureModuleInList(Type moduleType, IList`1 moduleList, List`1 requiredModules)
   at DevExpress.ExpressApp.ApplicationModulesManager.AddModuleIntoList(ModuleBase module, IList`1 moduleList, List`1 requiredModules)
   at DevExpress.ExpressApp.ModuleList.OnInsert(Int32 index, Object value)
   at System.Collections.CollectionBase.System.Collections.IList.Add(Object value)
   at DevExpress.ExpressApp.ApplicationModulesManager.AddModule(ModuleBase module)
----------InnerException----------
The type initializer for 'Xpand.Persistent.Base.General.XpandModuleBase' threw an exception.
----------StackTrace----------
   at Xpand.Persistent.Base.General.XpandModuleBase..ctor()
   at Xpand.ExpressApp.Email.EmailModule..ctor() in D:\a\1\s\Xpand\Xpand.ExpressApp.Modules\Email\EmailModule.cs:line 23
   at Create_EmailModule()
   at DevExpress.ExpressApp.Utils.Reflection.TypeData.CreateInstance(Object[] args)
----------InnerException----------
Could not find file 'C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\Common7\IDE\Xpand.Xpo.dll'.
----------StackTrace----------
   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share)
   at Mono.Cecil.ModuleDefinition.ReadModule(String fileName, ReaderParameters parameters)
   at Xpand.Persistent.Base.General.XpandModuleBase.AssemblyResolver.Resolve(AssemblyNameReference name) in D:\a\1\s\Xpand\Xpand.Persistent\Xpand.Persistent.Base\General\XpandModuleBase.cs:line 618
   at Mono.Cecil.MetadataResolver.Resolve(TypeReference type)
   at Mono.Cecil.TypeReference.Resolve()
   at Xpand.Extensions.Mono.Cecil.MonoCecilExtensions.<BaseClasses>d__0.MoveNext() in D:\a\1\s\src\Extensions\Xpand.Extensions.Mono.Cecil\BaseClasses.cs:line 9
   at System.Linq.Enumerable.Any[TSource](IEnumerable`1 source, Func`2 predicate)
   at Xpand.Extensions.Mono.Cecil.MonoCecilExtensions.IsSubclassOf(TypeDefinition childTypeDef, TypeDefinition parentTypeDef) in D:\a\1\s\src\Extensions\Xpand.Extensions.Mono.Cecil\IsSubclassOf.cs:line 6
   at Xpand.Persistent.Base.General.XpandModuleBase.<>c__DisplayClass99_0.<LoadAssemblyRegularTypes>b__1(TypeDefinition definition) in D:\a\1\s\Xpand\Xpand.Persistent\Xpand.Persistent.Base\General\XpandModuleBase.cs:line 631
   at System.Linq.Enumerable.WhereEnumerableIterator`1.MoveNext()
   at Xpand.Persistent.Base.General.XpandModuleBase.LoadAssemblyRegularTypes() in D:\a\1\s\Xpand\Xpand.Persistent\Xpand.Persistent.Base\General\XpandModuleBase.cs:line 632
   at Xpand.Persistent.Base.General.XpandModuleBase..cctor() in D:\a\1\s\Xpand\Xpand.Persistent\Xpand.Persistent.Base\General\XpandModuleBase.cs:line 111`

**Comment by apobekiaris** [comment]
I cannot tell much from the stack you posted the message is very abstracted. You either need to provide details or a sample that demonstrates the problem.

**Comment by TheArchbishop** [comment]
Hello, I have this exact issue after updating from DevExpress 19.2.7 to 20.1.4. I initially had the issue, mentioned elsewhere, that it was trying to load Mono.Cecil version 0.10.0.0, which I resolved by removing the Xamarin modules from my Visual Studio install. However, I now see this. 

Never had this before. Have been using Xpand since DevExpress 18.1, not seen this issue before. When I try and edit the model file, I see the exact same error as above, same line numbers and stack trace. Just as an experiment, I placed the Xpand.Xpo.Dll file in the folder it says, but made no difference, it still couldn't find the file !

The DLL does exist and is the correct version, in my module bin\debug folder.

Any ideas ?
